src/compileTranslations.ts

```diff
@@ -90,7 +89,10 @@ export default function compileTranslations(defaultLanguage: string) {
       }
 
       for (const language of languageTable.languages) {
-        const file = translationFiles.get(language)!;
```

Comment:
This is weird: if this return nothing, then the `try` block will fail.
Ok.
But then, at the `catch`, it uses `file.relative`, that would throw _again_ with the same runtime error